pkg/events/events.go

```diff
@@ -93,7 +93,12 @@ func New(object interface{}, eventType config.EventType, kind string) Event {
 	}
 
 	if kind != "events" {
-		event.Messages = []string{fmt.Sprintf("Resource %sd\n", eventType.String())}
+		switch eventType {
+		case config.ErrorEvent, config.InfoEvent:
+			event.Messages = []string{fmt.Sprintf("Resource %s\n", eventType.String())}
+		default:
+			event.Messages = []string{fmt.Sprintf("Resource %sd\n", eventType.String())}
+		}
```

Comment:
@aananthraj Why do you need switch case here if we are doing the same operation for every case?

Reply:
Have look at the `%s` and `%sd`
Events like `create` , `update` and `delete` comes with an extra `d` at the end as `Resource Created`
Where as `error`, `info` comes as `Resource Error`

Reply:
I guess its too small to notice 😅

Reply:
Aah, got it

Reply:
Probably better to add a comment on that